src/main/java/com/rmn/qa/aws/AwsVmManager.java

```diff
@@ -493,8 +493,11 @@ String getNodeConfig(final String uuid, final String hostName, final String brow
         Date createdDate = Calendar.getInstance().getTime();
 
         // Pass in the created date so we can know when this node was spun up
-        nodeConfig = nodeConfig.replaceAll("<CREATED_DATE>", AwsVmManager.NODE_DATE_FORMAT.format(createdDate));
-        nodeConfig = nodeConfig.replaceFirst("<HOST_NAME>", hostName);
+        String nodeCreatedDate = AwsVmManager.NODE_DATE_FORMAT.format(createdDate);
```

Comment:
Hmmm.  So the date is in quotes as seen in the node.linux.json file, and I can confirm this is working locally with Selenium 3.0.0.  Can you double check your issue to see if A.) Its still reproducing 3.4.0, and if so, B.) If its just something with Selenium 3.4.0 (try with 3.0.0 and see if it reproduces)